### PR TITLE
Add password visibility toggle in muc-enter-password dialog

### PR DESCRIPTION
**Pull request by harshitbansal05**
This pull request adds password visibility toggle in `muc-enter-password-dialog`. Also, it is disabled when the muc has a previous password.

### CODE REVIEW

File: src/main/java/eu/siacs/conversations/ui/XmppActivity.java [hunk 1]
```diff
@@ -715,17 +718,39 @@ private void quickEdit(final String previousValue,
...
 		if (hint != 0) {
-			editor.setHint(hint);
+			editorInputLayout.setHint(getResources().getString(hint));
 		}
 		editor.requestFocus();
 		editor.setText("");
+		if (password) {
+			editor.setInputType(InputType.TYPE_CLASS_TEXT
+					| InputType.TYPE_TEXT_VARIATION_PASSWORD);
+			if (!previousValue.equals("")) {
```
Comment (iNPUTmice) [inline src/main/java/eu/siacs/conversations/ui/XmppActivity.java]: can we use isEmpty() here?

Comment (harshitbansal05) [reply src/main/java/eu/siacs/conversations/ui/XmppActivity.java]: sure @iNPUTmice, do I need to do any more changes?

**Comment by harshitbansal05** [comment]
@iNPUTmice I have done the suggested change...

**Comment by whitetornado** [comment]
Is there something to set the app when inactive, Beside the screen lock, to unlock the app with a password?

**Pull request closed without merging**